src/Core/Services/Implementations/CipherService.cs

```diff
@@ -118,6 +121,21 @@ public class CipherService : ICipherService
                 }
                 else
                 {
+                    // Make sure the user can save new ciphers to their personal vault
+                    var userPolicies = await _policyRepository.GetManyByUserIdAsync(savingUserId);
+                    if (userPolicies != null)
+                    {
+                        foreach (var policy in userPolicies.Where(p => p.Enabled && p.Type == PolicyType.PersonalOwnership))
+                        {
+                            var org = await _organizationUserRepository.GetDetailsByUserAsync(savingUserId, policy.OrganizationId,
+                                OrganizationUserStatusType.Confirmed);
+                            if(org != null && org.Enabled && org.UsePolicies 
+                               && !(org.Type == OrganizationUserType.Admin || org.Type == OrganizationUserType.Owner))
```

Comment:
Would be an easier read here if you used something like this instead:
```c#
&& org.Type != OrganizationUserType.Admin && org.Type != OrganizationUserType.Owner
```
instead of the bang + grouping of OR conditions.

Reply:
Can for sure change it - I use DeMorgan's laws whenever possible since I learned it in College. :D

Reply:
Updated :)